doc/source/developer_guide/dg_tools.rst

```diff
@@ -0,0 +1,151 @@
+:tocdepth: 3
+
+.. _tools:
+
+Tools
+=============
+
+
+.. _cice4restart:
+
+CICE4 restart conversion
+-------------------------
+
+There is a Fortran program in **configuration/tools/cice4_restart_conversion**
+that will help convert a CICE4 restart file into a CICE5 restart file.
+There is a bit of documentation contained in that source code about how
+to build, use, and run the tool.  A few prognostic variables were changed
+from CICE4 to CICE5 which fundamentally altered the fields saved to
+the restart file.  See 
+**configuration/tools/cice4_restart_conversion/convert_restarts.f90** 
+for additional information.
+
+
+.. _jra55datasettool:
+
+JRA55 forcing datasets
+------------------------
+
+This section describes how to generate JRA55 forcing data for the CICE model.
+Raw JRA55 files have to be interpolated and processed into input files specifically
+for the CICE model.  A tool exists in **configuration/tools/jra55_datasets**
```

Comment:
note that this tool only works for grids supported by the CICE Consortium?

Reply:
I don't think we need to note that.  Any cice grid in netcdf format should be usable.